fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraResourceImpl.java

```diff
@@ -720,6 +681,31 @@ public Node getNodeVersion(final String label) {
 
     }
 
+    /**
+     * Helps ensure that there are no terminating slashes in the predicate.
+     * A terminating slash means ModeShape has trouble extracting the localName, e.g., for
```

Comment:
I have no idea why MODE has trouble with this. @osmandin worked that earlier ticket and he may know. My intention here was simply to use a jena parser (which can be more precise) rather than a regular expression (which was too greedy).